auto-instrumentation/okhttp/okhttp-3.0/testing/build.gradle.kts

```diff
@@ -3,16 +3,6 @@ plugins {
     id("net.bytebuddy.byte-buddy-gradle-plugin")
 }
 
-android {
-    buildTypes {
-        debug {
-            isMinifyEnabled = true
```

Comment:
Not sure if that's the reason because it seems to work well when the desugaring is disabled 🤔 The problem I saw with desugaring enabled is that it seems like the corelib classes are trimmed by R8, even though I add rules to keep them... Specifically, the test crashed because the function [Duration.toMillis](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#toMillis) wasn't found, and the error was mentioning the class like so: `Lj$/time/Duration;`. I'm guessing `j$` is the shade name given by Google's desugaring to the corelib classes.

Based on that, I tried to avoid it by adding these rules:

```txt
-keep class java.time.** { *; }
-keep class j$.time.** { *; }
```

But no luck, they seem to be ignored.

Reply:
Yes, it was done on API 33.

Reply:
Maybe try to enable [desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring) here ->
https://github.com/open-telemetry/opentelemetry-android/blob/main/auto-instrumentation/okhttp/okhttp-3.0/testing/build.gradle.kts